mesop/web/src/safe_iframe/safe_iframe.ts

```diff
@@ -5,28 +5,38 @@
 
 import {sanitizeJavaScriptUrl} from './sanitize_javascript_url';
 
-export function setIframeSrc(iframe: HTMLIFrameElement, src: string) {
+export function setIframeSrc(
+  iframe: HTMLIFrameElement,
+  src: string,
+  options: {allowSameOrigin: boolean},
+) {
   // Intentionally delegate to an impl function because the following
   // line will be modified downstream.
-  setIframeSrcImpl(iframe, src);
+  setIframeSrcImpl(iframe, src, options);
 }
 
 // copybara:strip_begin(external-only)
-function setIframeSrcImpl(iframe: HTMLIFrameElement, src: string) {
+function setIframeSrcImpl(
+  iframe: HTMLIFrameElement,
+  src: string,
+  options: {allowSameOrigin: boolean},
```

Comment:
Maybe add a comment about allowSameOrigin being an option. It was not immediately clear to me the reasoning.

Reply:
yep , i've added comments in this file and at each callsite.

Reply:
always allowing same origin since sandbox_iframe.html now renders all content itself in its own child sandboxed iframe that does _not_ have allow-same-origin